docker/webui/Dockerfile

```
@@ -1,10 +1,9 @@
+#!BuildTag: openqa_webui
 FROM opensuse/leap:15.2
 LABEL maintainer Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz <wnereiz@eienteiland.org>, Sergio Lindo Mansilla <slindomansilla@suse.com>
 LABEL version="0.3"
 
-RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
-    zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
-    zypper --gpg-auto-import-keys ref && \
+RUN zypper ar -f -G http://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 openQA && \
```

This is just minor suggestion, but `download.opensuse.org` is available via HTTPS too so we should maybe prefer it.
```suggestion
RUN zypper ar -f -G https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 openQA && \
```

---

to work within OBS, it needs to be http - and I actually can't see how https is preferred for this use case anyway.

---

OK. I was not aware of this. Thank you

---

@ilausuch  Does this need more clarification?

---

Well this is interesting, from the OBS perspective I agree with @coolo on the other hand for a local build could be more interesting to be https, but this can be easily managed by the user.
I resolved this conversation.